.github/pull_request_template.md

```diff
@@ -2,16 +2,28 @@
 
 ## Description
 
+<!-- Add a brief description of changes here -->
+
+## Screenshots (if appropriate):
+
+## How to Test
+
+<!-- Add steps to test this change. Include any other necessary relevant links -->
+
 <!--
 Replace PATH_TO_FEATURE with Vercel link
 -->
 
 :technologist: [Admin preview](PATH_TO_FEATURE)
 
-:rose: [Rose preview](PATH_TO_FEATURE)
-
 :desktop_computer: [Desktop preview](PATH_TO_FEATURE)
 
-<!-- Add a brief description of changes here. Include any other necessary relevant links -->
+## Checklist:
 
-### Screenshots (if appropriate):
+- [ ] I have performed a self-review of my code
```

Comment:
Is this necessary? Are people not self reviewing their own code before submitting it for PRs? I feel like the PR should be a draft if this is not true

Reply:
That is a good point, I mainly added this keeping interns and new engineers in mind. But happy to remove it if this seems unnecessary

Reply:
Yeah I personally feel this one sets the bar a little too low and if there are newer engineers, we can definitely make the expectation clear during onboarding since I feel like every other time this one won't be too useful

Reply:
removed!